### ISSUE TITLE
npm vs yarn

### ISSUE THREAD

**Issue by LeeCheneler** [rbrtsmith/nebula-css-react-starter#1]
Repo states you can use npm or yarn, but the repo already contains a `yarn.lock` file. Probably best to ditch it or only use yarn in readme. Users who use npm 5 would generate a lock file just for npm which should be committed, then they would end up with 2 lock lock files probably stipulating differing versions in places.

**Comment by rbrtsmith** [comment]
Yeah I get that.  I'm not sure which package manager to favour at this stage...
